### ISSUE TITLE
[Icon Request] Support for k

### ISSUE THREAD

**Issue by jshinonome** [vscode-icons/vscode-icons#2666]
- [x] I'm sure this issue is not a _duplicate_.
- [x] I'm submitting an icon request:
  - Type: `languageId`
  - Icon Name: `k`
  - Sample original Icon: N/A
  - Extensions: `.k`
  - Filenames: ``
  - Language ids: `k`
 
More info: https://estradajke.github.io/k9-simples/k9/index.html, https://shakti.sh/

**Comment by KingDarBoja** [comment]
I assume this is your extension: https://github.com/jshinonome/vscode-k9

**Comment by jshinonome** [comment]
Yes, that is my extension, still in progress. 
Don't think there is an official icon for k language.
Probably just something like this https://github.com/jshinonome/vscode-k9/blob/main/icon.svg

**Comment by jshinonome** [comment]
No need anymore, I will include this icon in my plugin.

**Comment by robertohuertasm** [comment]
Hey @jshinonome! Do you have the icon already? We can work with that! I'm reopening this until we see if it's worth closing it.

**Comment by jshinonome** [comment]
Thanks Robert. Latest vscode 1.64.0 allows me to add an icon for k language, so I will add an icon anyway.

**Comment by robertohuertasm** [comment]
I wasn't aware of that feature. Can you point me to the docs?

**Comment by robertohuertasm** [comment]
Is this icon still valid?  https://github.com/jshinonome/vscode-k9/blob/main/icon.svg I'll be glad to add it for this release.

**Comment by jshinonome** [comment]
> I wasn't aware of that feature. Can you point me to the docs?

https://code.visualstudio.com/updates/v1_64#_language-default-icons

**Comment by robertohuertasm** [comment]
Oh! That's a really nice feature!

**Comment by jshinonome** [comment]
> Is this icon still valid?  https://github.com/jshinonome/vscode-k9/blob/main/icon.svg I'll be glad to add it for this release.

Yeah, but we don't have an official icon for  k language, so it would be great if you could adjust it a bit.

**Comment by robertohuertasm** [comment]
Adjust it? What do you mean exactly? I can add it to our set according to our guidelines. Would that be ok?

**Comment by jshinonome** [comment]
Make it align with other icons. Thank you very much.

**Comment by robertohuertasm** [comment]
Ok, I will associate it to the language id you define in your extension.

**Comment by robertohuertasm** [comment]
@jshinonome done! It will be published in the coming release! thanks!
